app/Http/Controllers/FoodController.php

```diff
@@ -3,8 +3,73 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\Food;
+use App\Category;
+use App\Http\Requests\FoodPostRequest;
+use Image;
+use Session;
 
 class FoodController extends Controller
 {
-    //
+    protected $food;
+    
+    /**
+     * FoodController constructor.
+     *
+     * @param Food $food dependence injection
+     */
+    public function __construct(Food $food)
+    {
+        $this->food = $food;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $foods = $this->food->with('category')->paginate(10);
+        return view('foods.index', ['foods'=>$foods]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $categoryName = Category::orderBy('id', 'ASC')->get();
```

Comment:
By default `ORDER BY` already follow `id ASC`. You don't need to add `orderBy('id', 'ASC')` to order again.